packages/material-ui/src/styles/useThemeProps.d.ts

```diff
@@ -16,7 +12,7 @@ export interface AdditionalThemeProps<Theme> {
 }
 
 export default function useThemeProps<
-  Theme extends ThemeWithProps,
```

Comment:
@siriwatknp let's revert this change and try to use the generic in `useThemeProps`

Reply:
okay, done.